homeassistant/components/bmw_connected_drive/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      Does not have custom services
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
Please make sure every config flow test ends in either CREATE_ENTRY or ABORT to test that the flow is able to finish and recover from errors

Reply:
`test_authentication_error`, `test_api_error` and `test_connection_error` can be parametrized

Reply:
Fixed by #133704.